Sources/SourceKit/sourcekitd/SwiftLanguageServer.swift

```diff
@@ -1374,3 +1374,19 @@ extension sourcekitd_uid_t {
     }
   }
 }
+
+// FIXME: sourcekit only accepts `working-directry` through swift compiler args.
+// Injecting the arg here if workingDirectory presents in the settings,
+// but not in the actual compiler args.
+extension FileBuildSettings {
+  
+  func getSwiftCompilerArguments() -> [String] {
```

Comment:
I suggest something like `swiftCompilerArgumentsWithWorkingDirectory`.  Also, is this really a "FIXME" in the comment above? What would you do differently here?  I would work that comment into the documentation comment for this API.